finetune/model.py

```diff
@@ -101,17 +99,54 @@ def fp16_variable_getter(
     )
 
 
-def get_variable_getter(estimator_mode, features, fp16_predict):
-    if estimator_mode == tf.estimator.ModeKeys.PREDICT and fp16_predict:
+def mp_variable_getter(
```

Comment:
Could we name this to something other than mp so my brain stops reading "multiprocessing_variable_getter" haha